src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,76 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @AfterEach
+    void afterEach() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_uniqueLogin_Ok() {
+        User expected = new User("Bastard", "123456", 20);
+        User expected1 = new User("Bitch", "123456", 24);
+        registrationService.register(expected);
+        registrationService.register(expected1);
+        assertEquals(expected, Storage.people.get(0));
+        assertEquals(expected1, Storage.people.get(1));
+        assertEquals(2, Storage.people.size());
+    }
+
+    @Test
+    void register_nullLogin_NotOk() {
```

Comment:
```suggestion
    void register_nullLogin_notOk() {
```
the same for other similar cases